megamek/src/megamek/client/bot/princess/Precognition.java

```diff
@@ -277,7 +281,8 @@ private void pause() {
         while (!getWaiting().get() && !getDone().get()) {
             try {
                 Thread.sleep(100);
-            } catch (InterruptedException ignored) {
+            } catch (Exception ignored) {
+
```

Comment:
Just catching for the loop?

Reply:
It's a thread delay, so just ignoring the exception is fine here. I will be looking into any ignored exceptions once the full logging migration is done to see if we are ignoring things that actually make sense to log instead.

Reply:
Ahh. Thought as much.